python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_additional_paths.schema.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+type: dict
+$defs:
+  additional_paths:
+    type: dict
+    keys:
+      install:
+        type: bool
+      install_ecmp_primary:
+        type: bool
```

Comment:
Please add description for both `install` and `install_ecmp_primary`, also mention that it is applicable only for `router_bgp.vrfs.address-family ipv4.ipv6.bgp`.

Also you can modify this to -

```
install:
   type: dict
   keys:
      enabled:
          type: bool
      ecmp_primary:
          type: bool
```

Reply:
Descriptions of these commands differ a lot.

Install : Install BGP backup path.
Install_ecmp_primary: Allow additional path with ECMP primary path.